File: easy/python/1119.py
# https://leetcode.com/problems/remove-vowels-from-a-string/description/
# Time: O(n)
# Space: O(n)
class Solution:
    def removeVowels(self, s: str) -> str:
        non_vowels = []

        for char in s:
            if (
                char != "a"
                and char != "e"
                and char != "i"
                and char != "o"
                and char != "u"
            ):
                non_vowels.append(char)
        return "".join(non_vowels)

File: easy/ruby/1119.rb
# https://leetcode.com/problems/remove-vowels-from-a-string/
# @param {String} s
# @return {String}
def remove_vowels(s)
    vowels = {'a'=> 1, 'e'=> 1, 'i'=> 1, 'o'=> 1, 'u'=> 1}
    arr = []
    s.each_char do |char|
        if !vowels.has_key?(char)
            arr.push(char) 
        end       
    end
    arr.join("")
end